aztk_cli/spark/endpoints/cluster/cluster_create.py

```diff
@@ -40,6 +43,10 @@ def execute(args: typing.NamedTuple):
     # read cluster.yaml configuartion file, overwrite values with args
```

Comment:
(While you're here, could you fix the spelling of "configuration" in the comment?)